src/app/templates/documents.ftl

```diff
@@ -0,0 +1,49 @@
+<@layout>
+    <#macro styles>
+        <link rel="stylesheet" href="${appUrl}/css/style.min.css"/>
+        <style>
+            ul li {
```

Comment:
You want to leave this style here, or pull out into the scss?

Reply:
Ok I was being lazy, I'll move it